src/app/container/versions/versions.component.html

```diff
@@ -15,153 +15,124 @@
+      </th>
+      <td mat-cell *matCellDef="let version">
+        <span fxLayout="row" fxLayoutGap="0.5rem" fxLayoutAlign="start center">
+          <span *ngIf="defaultVersion === version.name" class="label label-primary">Default</span>
+          <!-- The href is so that the cursor changes to pointer on hover while not actually going anymore when clicked -->
+          <a
+            href="javascript:undefined"
+            *ngIf="selectedTag?.name !== version?.name"
+            matTooltip="Click to select version"
+            (click)="setVersion(version)"
+            >{{ version?.name }}</a
+          >
+          <span *ngIf="selectedTag?.name === version?.name">{{ version?.name }}</span>
+        </span>
+      </td>
+    </ng-container>
+    <ng-container matColumnDef="reference">
+      <th scope="col" mat-header-cell *matHeaderCellDef mat-sort-header matTooltip="">Git Reference</th>
+      <td mat-cell *matCellDef="let version">
+        <div *ngIf="tool?.mode === DockstoreToolType.ModeEnum.HOSTED">
+          {{ version?.versionEditor?.name }}
+        </div>
+        <div *ngIf="tool?.mode !== DockstoreToolType.ModeEnum.HOSTED">
+          <i *ngIf="version.referenceType === 'BRANCH'" class="fa fa-code-fork" aria-hidden="true" title="Branch"></i>
+          <i *ngIf="version.referenceType === 'TAG'" class="fa fa-tag" aria-hidden="true" title="Tag"></i>
+          <span
+            *ngIf="version.commitID && (version.commitID | commitUrl: tool?.providerUrl); else noCommitID"
+            [matTooltip]="version.commitID ? 'Commit ID: ' + version.commitID : ''"
+          >
+            &nbsp;<a [href]="version.commitID | commitUrl: tool?.providerUrl">{{ version.reference || 'n/a' }}</a>
+          </span>
+          <ng-template #noCommitID
+            ><span>{{ version.reference || 'n/a' }}</span></ng-template
+          >
+        </div>
+      </td>
+    </ng-container>
+    <ng-container matColumnDef="last_built">
+      <th scope="col" mat-header-cell *matHeaderCellDef mat-sort-header matTooltip="">Date Built</th>
+      <td mat-cell *matCellDef="let version">
+        {{ version.last_built | date: 'yyyy-MM-dd HH:mm' }}
+      </td>
+    </ng-container>
+    <ng-container matColumnDef="valid">
+      <th scope="col" mat-header-cell *matHeaderCellDef mat-sort-header matTooltip="">Valid</th>
+      <td mat-cell *matCellDef="let version">
+        <mat-icon *ngIf="version.valid">check</mat-icon>
+      </td>
+    </ng-container>
+    <ng-container matColumnDef="hidden">
+      <th scope="col" mat-header-cell *matHeaderCellDef mat-sort-header matTooltip="">Hidden</th>
+      <td mat-cell *matCellDef="let version">
+        <mat-icon *ngIf="version.hidden">check</mat-icon>
+      </td>
+    </ng-container>
+    <ng-container matColumnDef="verified">
+      <th
+        scope="col"
+        mat-header-cell
+        *matHeaderCellDef
+        matTooltip="A version is verified if it has been verified to work by a person/group."
```

Comment:
Maybe `A version is verified if it has been verified to work by a third party`

Reply:
This was carried over from before. It matches the workflow counterpart. I don't mind changing both of them right now, but it seems the suggestion is kind of inaccurate given that it's mostly me verifying them.

Reply:
I'm ok with it being aspirational. I think the main thing is just to convey the idea that it isn't the author themselves verifying it.  Whether we use the term "third party", "outside party" as here https://docs.dockstore.org/en/develop/advanced-topics/verification.html , I'm open to ideas. 

But currently it seems unclear to me.

Reply:
I changed the tooltip to what Solomon said.  Basically "if the column says verified platform, then the tooltip should also say it"